client/src/mergeUtils.js

```diff
@@ -248,20 +249,19 @@ export function areAllSet(...args) {
 export function getInfoButton(infoText) {
   return (
     <Tooltip2 content={infoText} intent={Intent.PRIMARY}>
-      <Button minimal icon={IconNames.INFO_SIGN} intent={Intent.PRIMARY} />
+      <Button variant="default">
+        <Icon icon={IconNames.INFO_SIGN} intent={Intent.PRIMARY} />
+      </Button>
     </Tooltip2>
   )
 }
 
 export function getClearButton(onClear) {
   return (
     <Tooltip2 content="Clear field value" intent={Intent.DANGER}>
-      <Button
-        icon={IconNames.DELETE}
-        outlined
-        intent={Intent.DANGER}
-        onClick={onClear}
-      />
+      <Button variant="outline-danger" onClick={onClear}>
+        <Icon icon={IconNames.DELETE} intent={Intent.DANGER} />
```

Comment:
Property intent should be removed because when button is hovered over, red icon on red background is barely visible.